src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1,71 +1,138 @@
 /*
-Copyright (c) 2008, Yahoo! Inc. All rights reserved.
-Code licensed under the BSD License:
-http://developer.yahoo.net/yui/license.txt
-version: 2.6.0
-*/
-html{color:#000;background:#FFF;}body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,code,form,fieldset,legend,input,button,textarea,p,blockquote,th,td{margin:0;padding:0;}table{border-collapse:collapse;border-spacing:0;}fieldset,img{border:0;}address,caption,cite,code,dfn,em,strong,th,var,optgroup{font-style:inherit;font-weight:inherit;}del,ins{text-decoration:none;}li{list-style:none;}caption,th{text-align:left;}h1,h2,h3,h4,h5,h6{font-size:100%;font-weight:normal;}q:before,q:after{content:'';}abbr,acronym{border:0;font-variant:normal;}sup{vertical-align:baseline;}sub{vertical-align:baseline;}legend{color:#000;}input,button,textarea,select,optgroup,option{font-family:inherit;font-size:inherit;font-style:inherit;font-weight:inherit;}input,button,textarea,select{*font-size:100%;}
-html, body {
-    background-color: #efefef;
-}
+    IMPORTANT: This is a Work-in-Progress and NOT FINISHED CSS file.
+               Please, don't review its contents yet.
+ */
+
+/*! Bootstrap normalize.css v3.0.3 | MIT License */
```

Comment:
normalize.css does not come from Bootstrap. It is a separate project (which is indeed used in Bootstrap): https://github.com/necolas/normalize.css